### PR TITLE
[FEAT]: Metadata SRC-9

### PR DESCRIPTION
**Pull request by JonGlazkov**
**What was made:**

- Changed the nomenclature of our metadata `keys` in UI to be according with `SRC-9`.
- Removed `NFT_Contract` Abi.
- Adjusted `TokenInfo` method in `SDK` and fixed tests for metadata.

### CODE REVIEW

File: packages/sdk/src/methods/token.ts [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 } from '../utils';
 
 type TokenInfo = {
-  name: string;
-  image: string;
-  symbol: string;
+  assetId: string;
   subId: string;
   contractId: string;
 };
```
Comment (luisburigo) [inline packages/sdk/src/methods/token.ts:15]: revert this change, this method should return all removed fields.

File: packages/sdk/src/methods/token.ts [hunk 2]
```diff
@@ -44,17 +45,10 @@ export async function tokenInfo(handle: string, params?: ProviderParams) {
     return undefined;
   }
 
-  const { value: tokenName } = await registry.functions
-    .name({ bits: ZeroBytes32 })
-    .get();
-  const { value: tokenSymbol } = await registry.functions
-    .symbol({ bits: ZeroBytes32 })
-    .get();
+  registry.functions.metadata({ bits: assetId }, 'image:png');
```
Comment (luisburigo) [inline packages/sdk/src/methods/token.ts:48]: this method was called but the return value was not captured, it should return the token image.

File: packages/sdk/src/methods/token.test.ts [hunk 3]
```diff
@@ -17,31 +24,28 @@ describe('Test token', () => {
...
-    expect(image).toBe(`https://assets.bako.id/${name}`);
-    expect(symbol).toBe('BNFT');
-    expect(subId).toBe(sha256(domainToBytes(name)));
+    //@ts-ignore
+    const { assetId, subId, contractId } = await tokenInfo(name);
+
+    expect(assetId).toBe(
+      getMintedAssetId(config.REGISTRY_CONTRACT_ID, domainHash),
+    );
+    expect(subId).toBe(domainHash);
```
Comment (luisburigo) [inline packages/sdk/src/methods/token.test.ts:41]: revert this change, the test should remain the same.

**Review by luisburigo — CHANGES_REQUESTED** [review]

File: packages/sdk/src/methods/token.test.ts [hunk 4]
```diff
@@ -17,31 +24,30 @@ describe('Test token', () => {
...
+    expect(token?.subId).toBe(domainHash);
+    expect(token?.contractId).toBe(config.REGISTRY_CONTRACT_ID);
   });
 
   it('should undefined values on token not minted', async () => {
     const name = randomName();
     const tokenResult = await tokenInfo(name, { provider });
+
     expect(tokenResult).toBeUndefined();
   });
```
Comment (luisburigo) [inline packages/sdk/src/methods/token.test.ts:52]: this test is failing because the validation of whether the token exists was removed

https://github.com/infinitybase/bako-id/blob/fd4ba0ea4e9566406c58fd80d0698d4a3d7495b2/packages/sdk/src/methods/token.ts#L47-L49

**Review by luisburigo — CHANGES_REQUESTED** [review]
**SDK**
image

**Tests Contract**
image

File: packages/sdk/src/methods/token.test.ts (hunk 4 above)
Comment (JonGlazkov) [reply packages/sdk/src/methods/token.test.ts:52]: I added again in the previous [commit](https://github.com/infinitybase/bako-id/pull/40/commits/27c5b3cc00c7db5fa89c155fa4bd2f3d921220c8)

**Review by luisburigo — APPROVED** [review]

**Review by luisburigo — APPROVED** [review]
